package.json

```diff
@@ -5,10 +5,11 @@
   "license": "Apache-2.0",
   "scripts": {
     "clean": "workspaces-run -- rm -rf dist -- rm -f tsconfig.tsbuildinfo",
-    "start": "npx nodemon --exec \"start-storybook -p 6006 --ci\" --watch packages/ts-transform/ -e tsx",
+    "start": "yarn build && npx nodemon --exec \"start-storybook -p 6006 --ci\" --watch packages/ts-transform/ -e tsx",
+    "start:prod": "NODE_ENV=production yarn start",
     "start:inspect": "npx nodemon --exec \"node --inspect-brk node_modules/.bin/start-storybook -p 6006 --ci\" --watch packages/ts-transform/ -e tsx",
-    "test": "jest",
-    "test:cover": "jest --collectCoverage",
+    "test": "yarn build && jest --no-cache",
```

Comment:
> Note: the cache should only be disabled if you are experiencing caching related problems. On average, disabling the cache makes Jest at least two times slower.

Reply:
You alright with that or is it a left over from some testing?

Reply:
yeah - when playing with the transformer it gets cached too easily and ends up being... annoying. because then i have to clear it (when I remember that that's probably the issue hehe)

i figured the easy solution for now is to just not cache it.